.env

```diff
@@ -1 +1,2 @@
-DATABASE_URL=postgres://politico:niyobobo@127.0.01:5432/politico_db
```

Comment:
You have private, sensitive details in this file, and for security reasons should not push your `.env` files.

Reply:
I have removed `.env` file from git.